test/common/all-schemas.spec.js

```diff
@@ -27,7 +27,7 @@ describe('all schema tests', () => {
     if (!schemas.hasOwnProperty(k)) continue;
 
     // skip "checkObjectTypes" for these dist files used as enums
-    if (['definitions', 'constants', 'vaMedicalFacilities', 'caregiverProgramFacilities', 'form1010cgCertifications'].includes(k)) continue;
+    if (['HLR-CREATE-REQUEST-BODY_V1', 'definitions', 'constants', 'vaMedicalFacilities', 'caregiverProgramFacilities', 'form1010cgCertifications'].includes(k)) continue;
```

Comment:
Draft-07 added explicit conditional schema evaluationI(if-then). Current version of Ajv JSON schema validator used by this repo for unit test does not support Draft-07 so I excluded the new schema from unit tests